### PR TITLE
fix CommonJS interop section in modules doc

### PR DESCRIPTION
**Pull request by teppeis**
Output of default exports was changed since v5.0.

### CODE REVIEW

File: docs/usage/modules.md [hunk 1]
```diff
@@ -103,14 +103,15 @@ var bar = require("foo").foo;
 #### Interop
 
 In order to encourage the use of CommonJS and ES6 modules, when exporting a default
-export with **no** other exports `module.exports` will be set instead of `exports.default`.
+export with **no** other exports `module.exports` will be set in addtion to `exports["default"]`.
```
Comment (sebmck) [inline docs/usage/modules.md]: addtion -> addition

Comment (teppeis) [inline docs/usage/modules.md]: @sebmck oops... sorry, fixed.

**Comment by sebmck** [comment]
Thanks @teppeis!

**Comment by teppeis** [comment]
@sebmck thanks!

**Pull request merged**